nanshe_workflow.recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
     - bokeh >=0.12.4
     - future
     - webcolors
+    - nbconvert
     - notebook
```

Comment:
Seems the `notebook` also pulls in `nbconvert`. Still it should be an explicit dependency.